pyo3-macros-backend/src/pymethod.rs

```diff
@@ -653,7 +662,15 @@ pub fn impl_py_getter_def(
         }
     };
 
+    let mut cfg_attrs = TokenStream::new();
+    if let PropertyType::Descriptor { field, .. } = &property_type {
+        for attr in field.attrs.iter().filter(|attr| attr.path.is_ident("cfg")) {
+            attr.to_tokens(&mut cfg_attrs);
+        }
+    }
```

Comment:
I suspect it probably works to use an `Option` instead of having the intermediate `TokenStream` buffer:

```suggestion
    let mut cfg_attrs = None;
    if let PropertyType::Descriptor { field, .. } = &property_type {
        cfg_attrs = Some(field.attrs.iter().filter(|attr| attr.path.is_ident("cfg")));
    }
```

Reply:
I don't think I can `collect` into a tokenstream :(